### PR TITLE
Adds information about custom .sqlfluff and .sqlfluffignore files

### PR DESCRIPTION
**Pull request by will-sargent-dbtlabs**
## What are you changing in this pull request and why?
The current docs are a bit unclear about the fact that we now support custom `.sqlfluff` and `.sqlfluffignore` files.  This PR adds that information, as well as links to docs and resources to configure them.

This changes comes from responding to customer questions.

### CODE REVIEW

File: website/docs/guides/best-practices/how-we-style/2-how-we-style-our-sql.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ id: 2-how-we-style-our-sql
 ## Basics
 
 - ☁️ Use [SQLFluff](https://sqlfluff.com/) to maintain these style rules automatically.
-  - Reference this [SQLFluff config file](https://github.com/dbt-labs/jaffle-shop-template/blob/main/.sqlfluff) for the rules we use.
+  - Custom `.sqlfluff` configuration files are supported.
```
Comment (mirnawong1) [inline website/docs/guides/best-practices/how-we-style/2-how-we-style-our-sql.md]:
```suggestion
  - Customize `.sqlfluff` configuration files to your needs.
```

File: website/docs/guides/best-practices/how-we-style/2-how-we-style-our-sql.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ id: 2-how-we-style-our-sql
 ## Basics
 
 - ☁️ Use [SQLFluff](https://sqlfluff.com/) to maintain these style rules automatically.
-  - Reference this [SQLFluff config file](https://github.com/dbt-labs/jaffle-shop-template/blob/main/.sqlfluff) for the rules we use.
+  - Customize `.sqlfluff` configuration files to your needs.
+  - Reference this [SQLFluff config file](https://github.com/dbt-labs/jaffle-shop-template/blob/main/.sqlfluff) for the rules we use in our own projects.
```
Comment (mirnawong1) [inline website/docs/guides/best-practices/how-we-style/2-how-we-style-our-sql.md]:
```suggestion
  - Refer to our [SQLFluff config file](https://github.com/dbt-labs/jaffle-shop-template/blob/main/.sqlfluff) for the rules we us in our own projects. 

```

File: website/docs/guides/best-practices/how-we-style/2-how-we-style-our-sql.md [hunk 3]
```diff
@@ -6,7 +6,10 @@ id: 2-how-we-style-our-sql
 ## Basics
 
 - ☁️ Use [SQLFluff](https://sqlfluff.com/) to maintain these style rules automatically.
-  - Reference this [SQLFluff config file](https://github.com/dbt-labs/jaffle-shop-template/blob/main/.sqlfluff) for the rules we use.
+  - Customize `.sqlfluff` configuration files to your needs.
+  - Refer to our [SQLFluff config file](https://github.com/dbt-labs/jaffle-shop-template/blob/main/.sqlfluff) for the rules we us in our own projects. 
+
+  - We also support ignoring files and folders using a standard `.sqlfluffignore` file. Reference these docs for the syntax: [.sqlfluffignore syntax docs](https://docs.sqlfluff.com/en/stable/configuration.html#id2)
```
Comment (mirnawong1) [inline website/docs/guides/best-practices/how-we-style/2-how-we-style-our-sql.md]:
```suggestion
  - Exclude files and directories by using a standard `.sqlfluffignore` file. Learn more about the syntax in the [.sqlfluffignore syntax docs](https://docs.sqlfluff.com/en/stable/configuration.html#id2).
```

**Comment by will-sargent-dbtlabs** [comment]
@mirnawong1 At the risk of expanding the PR a bit too much, I just realized we also have a whole other tree where mentioning sqlignore might be valuable:
https://docs.getdbt.com/docs/cloud/dbt-cloud-ide/lint-format

I'd enter a tentative edit, but not quite sure where to fit it.. Inline, or FAQ's at the bottom?

File: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md [hunk 4]
```diff
@@ -63,7 +63,7 @@ With the dbt Cloud IDE, you can seamlessly use [SQLFluff](https://sqlfluff.com/)
 
 ### Customize linting
 
-SQLFluff is a configurable SQL linter, which means you can configure your own linting rules instead of using the default linting settings in the IDE.  
+SQLFluff is a configurable SQL linter, which means you can configure your own linting rules instead of using the default linting settings in the IDE. You can exclude files and directories by using a standard `.sqlfluffignore` file. Learn more about the syntax in the [.sqlfluffignore syntax docs](https://docs.sqlfluff.com/en/stable/configuration.html#id2).
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/lint-format.md:66]: hey @will-sargent-dbtlabs ! we thought of the same thing and added your bit here - wdyt?

File: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md [hunk 5]
```diff
@@ -122,6 +122,8 @@ capitalisation_policy = lower
 group_by_and_order_by_style = implicit
 ```
 </details>
+
+Refer to [How we style our SQL](/guides/best-practices/how-we-style/2-how-we-style-our-sql) for more info.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]:
```suggestion
For more info on styling best practices, refer to [How we style our SQL](/guides/best-practices/how-we-style/2-how-we-style-our-sql).
```

**Review by mirnawong1 — APPROVED** [review]
🚢

File: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md (hunk 4 above)
Comment (will-sargent-dbtlabs) [reply website/docs/docs/cloud/dbt-cloud-ide/lint-format.md:66]: Yeah, great stuff!

Comment (mirnawong1) [reply website/docs/docs/cloud/dbt-cloud-ide/lint-format.md:66]: thanks WIll!

**Pull request merged**